src/utilities/activityContent.ts

```diff
@@ -0,0 +1,63 @@
+import path from "path";
```

Comment:
path?

Reply:
This was already in the file (this change is mostly a filename that git misinterpreted). I think webpack is polyfiling this node package.